### ISSUE TITLE
[BUG]: AEBN search issue

### ISSUE THREAD

**Issue by joca879** [CodyBerenson/PGMA-Modernized#318]
# Mandatory #

## Describe the bug: ## 
Not able to match a title that is on AEBN site, namely the movie "2 x 10" from Bijou Studios.
It gives this error message in the log "ERROR (utils:5484) - AEBN - SEARCH:: ❌ File: __init__.py, Line: 279, Function: _process_function_call -> _search -> search"

## Specific Agent(s) Causing the Issue: ##
AEBN

## Index Site URL(s) Attempting to Match: ##
https://gay.aebn.com/gay/search?queryType=Free+Form&query=2%20x%2010

## Log Attached: ##
[com.plexapp.agents.AEBN.log](https://github.com/user-attachments/files/16822526/com.plexapp.agents.AEBN.log)

# Optional #

**Comment by joca879** [comment]
No one else has had a similar issue with AEBN agent search?

**Comment by fourstix** [comment]
Hi,
I'm not sure why it doesn't match on AEBN but I have this same movie named as "(Paladin) - 2x10" (different studio and without spaces in the name) so that it matches on GEVI.

**Comment by joca879** [comment]
Unfortunately I have other movies that are not in GEVI and AEBN gives the same error when I try to match. 
AEBN must have changed something in their site and now the search on the PGMA agent does not work

**Comment by fourstix** [comment]
Are you in the US in one of the states that passed an age verification law?  I had to set up a VPN to get the AEBN and other agents to work for me.

The details are in this discussion item.
https://github.com/CodyBerenson/PGMA-Modernized/discussions/300

**Comment by JPH71** [comment]
Just looking at this and its because AEBN has implemented an age verification step that effs eveything up.
As per discussion #300 is there anyone who has done this for Windows? I am rather clueless when it comes to VPNs

In regards to this age verification block, I am trying to set cookies that would fool the page and allow one to process as if age verification had been accepted.... no luck yet.....

**Comment by fourstix** [comment]
If it's easier to code and maintain, most of the sites with age verification will accept a userid / password. I would be okay with having the agent log in to the site with useird/ password parameters one supplies in the agent configuration.  (Then it would be a legit session request to the site, rather than a hack.)

**Comment by joca879** [comment]
> Are you in the US in one of the states that passed an age verification law? I had to set up a VPN to get the AEBN and other agents to work for me.
> 
> The details are in this discussion item. #300

I'm not in the US. The other agents work fine

**Comment by joca879** [comment]
> Just looking at this and its because AEBN has implemented an age verification step that effs eveything up. As per discussion #300 is there anyone who has done this for Windows? I am rather clueless when it comes to VPNs
> 
> In regards to this age verification block, I am trying to set cookies that would fool the page and allow one to process as if age verification had been accepted.... no luck yet.....

Yes, in the last days I noticed that when trying to enter AEBN it pops up a age verification form... damn it... :(

**Comment by JPH71** [comment]
I will try the user + password thing... good idea
